tests/tet/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8.12)
+INCLUDE(../setup_testsubproject.cmake)
+PROJECT(testsuite CXX)
+DEAL_II_PICKUP_TESTS()
```

Comment:
I would do
```
IF (DEAL_II_WITH_TET_SUPPORT)
  DEAL_II_PICKUP_TESTS()
ENDIF()
```

Reply:
but this can be a followup when you actually add useful tests here.

Reply:
I thought about this. But, I find it too restrictive, since there will be tests which will work with the default configuration.